pages/api/auth/[...nextauth].js

```diff
@@ -66,8 +44,9 @@ export default NextAuth({
     async signIn(user, account, profile) {
       let isAllowedToSignIn = false;
 
-      user.organizations.forEach((org) => {
-        if (org.slug === process.env.ORG_SLUG) {
+      let authorizedEmails = process.env.AUTHORIZED_EMAILS.split(',');
+      authorizedEmails.forEach((authorizedEmail) => {
+        if (user.email === authorizedEmail) {
```

Comment:
I think the better thing to do would be for `AUTHORIZED_EMAILS` to be more like `AUTHORIZED_EMAIL_DOMAINS`, and then we split on the `@` here and look for matches. That way, if an org gets a new employee/email address, we don't have to go in and update an env var.

Reply:
Oh cool - I wasn't sure if we could allow entire domains. Sounds good to me. I see you made the change already, thanks!